src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,7 +1,41 @@
-import React from 'react';
+import React, { useState } from 'react';
+import cn from 'classnames';
+import { getMovie } from '../../api';
+import { Movie } from '../../types/Movie';
+import { MovieCard } from '../MovieCard';
 import './FindMovie.scss';
 
-export const FindMovie: React.FC = () => {
+type Props = {
+  addMovie: (movie: Movie | null) => void;
+};
+
+export const FindMovie: React.FC<Props> = ({ addMovie }) => {
+  const [query, setQuery] = useState('');
+  const [movie, setMovie] = useState<Movie | null>(null);
+  const [error, setError] = useState(false);
+  const [loading, setLoading] = useState(false);
```

Comment:
use naming convention for the boolean variables: 
There is a convention to prefix boolean variables and function names with "is" or "has"